justfile

```diff
@@ -1,8 +1,19 @@
+build_target := if os_family() == "windows" {
+  "pc-windows-msvc"
+} else {
+  if os() == "macos" {
+    "apple-darwin"
+  } else {
+    "unknown-linux-gcc"
```

Comment:
```suggestion
    "unknown-linux-gnu"
```

Typo! my bad.

Reply:
Thanks 🙏 took me ages to figure out the macOS config, I didn't actually check the linux part!